include/crypto/riscv_arch.h

```diff
@@ -56,4 +56,8 @@ static const size_t kRISCVNumCaps =
 # include "riscv_arch.def"
 ;
 
+/* Extension combination tests. */
+#define RISCV_HAS_ZBKB_AND_ZKND_AND_ZKNE() (RISCV_HAS_ZKNB() && RISCV_HAS_ZKND() && RISCV_HAS_ZKNE())
```

Comment:
Is it expected that the extension is called `ZBKB` in the macro name but seems to rely on `RISCV_HAS_ZKNB()` ?

Reply:
Good catch! The name is correct, but the test is not.
I will fix that!